invokeai/backend/grounded_sam/grounding_dino_pipeline.py

```diff
@@ -0,0 +1,36 @@
+from typing import Optional
+
+import torch
+from PIL import Image
+from transformers.pipelines import ZeroShotObjectDetectionPipeline
+
+from invokeai.backend.grounded_sam.detection_result import DetectionResult
+
+
+class GroundingDinoPipeline:
```

Comment:
Maybe this folder needs to be moved in to `backend>image_util` to be consistent with other image related stuff we have.

Reply:
Done in [9f448fe](https://github.com/invoke-ai/InvokeAI/pull/6701/commits/9f448fecb7cc8bb07a7741ee898294661fc92cc9)